wrims-install/src/main/resources/installer/.eclipseproduct

```diff
@@ -1,3 +1,3 @@
 name=WRIMS
-id=dwr.wrims.gui
+id=water.wrims.gui
```

Comment:
Is this change correct? without gov.ca, water loses context.

Reply:
unable to verify in corresponding spots, because this string matches all over the project